src/main/java/gg/uhc/uhc/modules/health/HealthCommand.java

```diff
@@ -78,15 +78,15 @@ public HealthCommand(MessageTemplates messages, double maximumHealthLimit) {
                 .acceptsAll(ImmutableList.of("a", "all"), "Check/modify all online players");
 
         setCurrentHealthSpec = parser
-                .accepts("set", "Set the health of players")
+                .accepts("s", "Set the health of players")
```

Comment:
You've misunderstood here, you can use both a short and long flag, check out the other commands. `ImmutableSet.of('s', 'set')` + max